selectparse/selectparse.go

```diff
@@ -189,10 +189,10 @@ func extractFromKeywordsOrFunc(column string) string {
 		buffer = append(buffer, string(r))
 	}
 	if len(buffer) != 0 && depth == 0 {
-		return strings.Trim(strings.Join(buffer, ""), " ")
+		return extractFromSingleWord(strings.Trim(strings.Join(buffer, ""), " "))
```

Comment:
I am not necessarily disagreeing, but why?

Reply:
Because without the test fails because it results in e.g. `table.id` instead of just `id` based on the input of `DISTINCT table.id`.